### PR TITLE
[TW-827] Doc on-prem Git integration changes for GitLab and Azure

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good!

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Keep in mind the following when connecting to a repository:
 
 * **For GitHub connections, there's a limit of ten auth tokens per user per application imposed by GitHub.** If you create more than ten connections with the same user, the additional tokens will be revoked in the order that they were created. Teams can use other Postman accounts to create more than ten integrations.
 
-    > If you used the earlier standalone GitHub integration for two-way sync of an API definition, you must delete the previous integration to connect a repository to your API. An existing integration will continue to function, but you can't add new integrations for two-way sync to an API definition.
+    > If you used the earlier standalone GitHub integration for two-way sync of an API definition, you must delete the existing integration to connect a repository to your API. An existing integration will continue to function, but you can't add new integrations for two-way sync to an API definition.
```
Comment (shashankawasthi88) [inline src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:73]: @mark-anthony-postman - I don't get the rationale behind adding this. Why was this added at first place

**Review by shashankawasthi88 — APPROVED** [review]
Looks good

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:73]: @shashankawasthi88 I'm not sure of the reasoning either! 😄 I went ahead and removed the note.

File: src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md [hunk 2]
```diff
@@ -93,14 +93,7 @@ You can connect to a GitHub, Bitbucket, GitLab, or Azure DevOps repository hoste
...
 
-<!--
-> Use the [Postman desktop app](/docs/getting-started/installation-and-updates/) to connect to an on-premises Git repository. You can't use the [Postman web app](https://learning.postman.com/docs/getting-started/installation-and-updates/#using-the-postman-web-app).-->
-
-> **Some on-premises repositories require static IP support.** For GitLab Self-Managed and Azure DevOps Server, your repository must be able to reach the IP address mentioned in [static IP support](/docs/integrations/intro-integrations/#static-ip-support). Contact your IT team to allowlist the appropriate IP address. Static IP support isn't required for GitHub Enterprise Server.
-
-<!-- -->
-
-> **Use the Postman desktop app to connect to GitHub Enterprise Server.** You can't use the Postman web app to connect to an on-premises GitHub repository.
+> **Use the [Postman desktop app](/docs/getting-started/installation-and-updates/) to connect to GitHub Enterprise Server, GitLab Self-Managed, or Azure DevOps Server.** You can't use the [Postman web app](https://learning.postman.com/docs/getting-started/installation-and-updates/#using-the-postman-web-app) to connect to an on-premises GitHub repository.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/versioning-an-api/using-external-git-repo.md:96]: Thanks! Per Shashank, we'll wait until the changes for Azure DevOps Server release, and then we'll push out this PR to prod.
